reference/supportability.md

```diff
@@ -0,0 +1,159 @@
+## Supportability
+
+The supportability tool collects Mayastor specific information from the cluster using the [kubectl plugin](https://mayastor.gitbook.io/introduction/v/develop/reference/kubectl-plugin) command-line tool. It uses the dump command, which interacts with the Mayastor services to build an archive (ZIP) file that acts as a placeholder for the bundled information.
+
+
+### Using the supportability tool
+
+To bundle Mayastor's complete system information, execute:
+
+{% tab title="Command" %}
+```text
+kubectl mayastor dump system -n mayastor -d <output_directory_path>
+```
+{% endtab %}
+
+{% hint style="info" %}
+**Note:** If the nodes exist on a private network or the node ports are not exposed to an outside cluster, all the above commands need to pass these additional parameters:
+kubectl mayastor dump <resource> -d <output_directory> -r http://127.0.0.1:30011 -l http://127.0.0.1:3100 -e http://127.0.0.1:2379 -n mayastor
+{% endhint %}
+
+
+To view all the available options and sub-commands that can be used with the dump command, execute:
+
+{% tabs %}
+{% tab title="Command" %}
+```text
+kubectl mayastor dump --help
+```
+{% endtab %}
+{% tab title="Sample Output" %}
+```text
+kubectl-mayastor-dump 
+`Dump` resources
```

Comment:
Is line 33 a part of the output? Feels a bit out-of-place or incomplete.
cc @Abhinandan-Purkait

Reply:
```
kubectl-mayastor-dump 
`Dump` resources

USAGE:
    kubectl-mayastor dump [OPTIONS] <SUBCOMMAND>

OPTIONS:
    -d, --output-directory-path <OUTPUT_DIRECTORY_PATH>
            Output directory path to store archive file [default: ./]

    -e, --etcd-endpoint <ETCD_ENDPOINT>
            Endpoint of ETCD service, if left empty then will be parsed from the internal service
            name

    -h, --help
            Print help information

    -j, --jaeger <JAEGER>
            Trace rest requests to the Jaeger endpoint agent

    -k, --kube-config-path <KUBE_CONFIG_PATH>
            Path to kubeconfig file

    -l, --loki-endpoint <LOKI_ENDPOINT>
            Endpoint of LOKI service, if left empty then it will try to parse endpoint from Loki
            service(K8s service resource), if the tool is unable to parse from service then logs
            will be collected using Kube-apiserver

    -n, --namespace <NAMESPACE>
            Kubernetes namespace of mayastor service, defaults to mayastor [default: mayastor]

    -o, --output <OUTPUT>
            The Output, viz yaml, json [default: none]

    -r, --rest <REST>
            The rest endpoint to connect to

    -s, --since <SINCE>
            Period states to collect all logs from last specified duration [default: 24h]

    -t, --timeout <TIMEOUT>
            Specifies the timeout value to interact with other modules of system [default: 10s]

SUBCOMMANDS:
    etcd       Collects information from etcd
    help       Print this message or the help of the given subcommand(s)
    node       Collects information about particular node matching to given node ID
    nodes      Collects information about all nodes
    pool       Collects information about particular pool and its descendants matching to given
                   pool ID
    pools      Collects information about all pools and its descendants (nodes)
    system     Collects entire system information
    volume     Collects information about particular volume and its descendants matching to
                   given volume ID
    volumes    Collects information about all volumes and its descendants (replicas/pools/nodes)

Supportability - collects state & log information of services and dumps it to a tar file.
```

This is how it looks currently.